events/event/forms.py

```diff
@@ -20,20 +21,35 @@ def validate(self, value):
 class EventCreateForm(forms.ModelForm):
     class Meta:
         model = Event
-        exclude = ['participants', 'start_date', 'end_date']
+        exclude = ['participants', 'start_date', 'end_date', 'owner']
     start_date = DateValidator()
     end_date = DateValidator()
 
+    def clean_title(self):
+        title = self.cleaned_data['title']
+        if len(title) > 20:
+            raise forms.ValidationError("Cannot be more than 20 characters")
+        return title
+    
+    def clean_description(self):
+        description = self.cleaned_data['description']
+        if len(description) > 200:
+            raise forms.ValidationError("Cannot be more than 200 characters")
+        return description
+
     def clean_location(self):
-        data = self.cleaned_data.get('location')
+        location = self.cleaned_data['location']
         try:
-            isinstance(data[0], float) and isinstance(data[1], float)
+            isinstance(location[0], float) and isinstance(location[1], float)
         except:
             raise forms.ValidationError("Enter valid data")
-        return data
-    
+        return location
+
+    def clean_start_date(self):
+        return TZ.localize(datetime.utcfromtimestamp(float(self.cleaned_data['start_date']))) 
+
     def clean_end_date(self):
-        start_date = self.cleaned_data.get('start_date')
-        end_date = self.cleaned_data.get('end_date')
-        if float(end_date) - float(start_date) < 60 * 15:
-            raise forms.ValidationError("End date cannot be earlier than Start date")
+        end_date = TZ.localize(datetime.utcfromtimestamp(float(self.cleaned_data['end_date']))) 
+        if (end_date - self.cleaned_data['start_date']).seconds < 60 * 15:
```

Comment:
60 * 15 is called magic numbers. It is recommended to use fine named constants or at least add comment what this math magic means.